app/services/service_offering/add_to_portfolio_item.rb

```diff
@@ -1,6 +1,7 @@
 module ServiceOffering
   class AddToPortfolioItem
-    IGNORE_FIELDS = %w(id created_at updated_at portfolio_id).freeze
+    SERVICE_OFFERING_FIELDS = %w(name description display_name long_description
```

Comment:
Just a quick nit - going to need the `service_offering_icon_id` as well. Or I can add it to my PR since I added the icon column, that would probably make more sense.